apps/web/src/app/layout.tsx

```diff
@@ -53,15 +50,11 @@ export default function RootLayout({
           closeButton={ToastCloseButton}
           icon={ToastIcon}
         />
-        <div className="max-w-6xl w-full mx-auto min-h-screen flex flex-col justify-between pt-[64px] px-[25px]">
+        <div className="max-w-6xl min-h-screen w-full mx-auto flex flex-col justify-between">
           <AnalyticsProvider>
             <SupabaseAuthProvider>
               <QueryProvider>
-                <GlobalModalProvider>
-                  <Header />
-                  <MainWrapper>{children}</MainWrapper>
-                  <Footer />
-                </GlobalModalProvider>
```

Comment:
please make sure you apply header/footer to other existing layouts, this page fails:

https://darklake-git-lewis-dar-71-webapp-add-mev-chec-7883a6-darklakefi.vercel.app/results/B3C654sBLMk7FgELgdtQJq4u39dC5ejLJpi62dnKn7cg

Reply:
Sorted